contracts/Comptroller/Diamond/facets/PolicyFacet.sol

```diff
@@ -57,6 +78,7 @@ contract PolicyFacet is IPolicyFacet, XVSRewardsHelper {
      */
     // solhint-disable-next-line no-unused-vars
     function mintVerify(address vToken, address minter, uint256 actualMintAmount, uint256 mintTokens) external {
+        ensureUserAllowed(minter);
```

Comment:
I think this is not needed, because we are already checking the permission for minting in the `mintAllowed` check (the "pre-hook")